### PR TITLE
package/linux-mainline: Bump to 6.3

### PR DESCRIPTION
**Pull request by alistair23**
Bump to the 6.3 kernel and update the list of issues

### CODE REVIEW

**Comment by Eeems** [comment]
For the otg control, are you planning on importing the remarkable pogo modules in the future?

**Comment by alistair23** [comment]
I'm not currently planning on it, I have no interest in it and I feel it will be difficult to upstream. If someone else wanted to they are welcome to though

**Comment by Eeems** [comment]
> I'm not currently planning on it, I have no interest in it and I feel it will be difficult to upstream. If someone else wanted to they are welcome to though

There wouldn't be too much of a need to upstream it to be honest, it would be simple enough to just add to this package, just to resolve one of the known issues being shipped with toltec.

**Comment by Eeems** [comment]
@alistair23 sorry for how long this is taking. I don't have a rM2 to test with, and I'm waiting on the other maintainers to do so.

File: package/linux-mainline/package [hunk 1]
```diff
@@ -6,7 +6,7 @@ archs=(rm2)
 pkgnames=(linux-mainline)
 pkgdesc="reMarkable 2 kernel based on the mainline kernel"
 url=https://www.kernel.org
-pkgver=6.2.0-2
+pkgver=6.3.0-1
 timestamp=2022-05-22T21:50:09Z
```
Comment (Eeems) [inline package/linux-mainline/package]: I just noticed that the timestamp wasn't updated

**Comment by Eeems** [comment]
I have a rM2 to test with now, although I don't have the parts for recovery quite yet. That said, I'll probably still be able to test. I'll test on both 2.15, and 3.3 with a folio.

**Review by Eeems — CHANGES_REQUESTED** [review]
The package is missing the installdepends on kernelctl.

**Comment by jkesselm** [comment]
If I'd known you didn't, I'd have been persuadable to lend mine...

--
   /_  Joe Kesselman (he/him/his)
-/ _) My Alexa skill for New Music/New Sounds fans:
   /   https://www.amazon.com/dp/B09WJ3H657/

Caveat: Opinionated old geezer with overcompensated writer's block. May be redundant, verbose, prolix, sesquipedalian, didactic, officious, or redundant.
________________________________

**Comment by Etn40ff** [comment]
6.3 works on 2.15 but 6.2 no. 

```
reMarkable: ~/ cd /tmp/
reMarkable: /tmp/ tar xf /tmp/linux-mainline_6.3.0-1_rm2.ipk 
reMarkable: /tmp/ tar xf /tmp/data.tar.gz 
reMarkable: /tmp/ tar xf ./opt/usr/share/kernelctl/mainline-6.3.0.tar.bz2
reMarkable: /tmp/ cp -r /tmp/lib/modules/6.3.0/ /lib/modules/
reMarkable: /tmp/ kexec -l /tmp/boot/zImage --command-line="console=ttymxc0,115200 root=/dev/mmcblk2p2 rootwait rootfstype=ext4 rw quiet panic=20 systemd.crash_reboot systemd.show_status=0 loglevel=0 crashkernel=32M" --dtb=/tmp/boot/zero-sugar.dtb 
reMarkable: /tmp/ /bin/sync
reMarkable: /tmp/ systemctl kexec
```

**Comment by Eeems** [comment]
I'll have to test on 3.3 to confirm that it works there as well, and then adjust the package recipe to allow it in os3 again.

**Comment by alistair23** [comment]
> The package is missing the installdepends on kernelctl.

Thanks for adding it!

**Comment by Eeems** [comment]
Works on 2.15 with expected issues. Next will be to test on 3.3 with the folio.

**Comment by Eeems** [comment]
Okay, this works on 3.3, I'll update the PR to have it no longer be removed and to add the note that the folio doesn't work (as it doesn't).

**Review by Eeems — APPROVED** [review]

**Pull request merged**